auctioneerrpc/auctioneer.proto

```diff
@@ -1140,6 +1140,12 @@ message TermsResponse {
     the duration buckets currently are in.
     */
     map<uint32, DurationBucketState> lease_duration_buckets = 8;
+
+    /*
+    The value used by the auctioneer to determine if an account expiry height
+    needs to be extended after participating in a batch and for how long.
+    */
+    uint32 auto_renew_extension_blocks = 9;
```

Comment:
We should probably also add this to the client side `NextBatchInfoResponse` so the user can see the value somewhere.